### ISSUE TITLE
[Request] Spotify URI

### ISSUE THREAD

**Issue by finnbrue** [fabiofalci/sconsify#80]
Is it possible to include searching using Spotify URIs or maybe play them only with a command? (Maybe with sconsify -command url="")

**Comment by fabiofalci** [comment]
what do you mean by `searching using Spotify URIs`?

**Comment by finnbrue** [comment]
Every Song, Album or Playlists has his own URI like spotify:track:0dHWp33vt26Z7i06ljKyg0 . I mean playing a song with this URI via command.

**Comment by fabiofalci** [comment]
Yeah, should be easy, as we pass those URIs to Spotify when playing a track.

Actually, if I remember correctly, one libspotify C example does exactly that: you pass an URI by command line and then it plays the track.

The problem is, libspotify is not available anymore, and I don't have plans for sconsify after Spotify removed libspotify binaries from their website. Also, I don't think they will offer any good replacement for libspotify.

**Comment by finnbrue** [comment]
Okay, thanks for the information. Is there a reason why they stopped developing libspotify? Can i get the library from somewhere or is it no possible?

**Comment by fabiofalci** [comment]
I don't know why they have stopped development, no announcement or anything. 
You can get an idea of what has been happening here:
https://jodal.no/2016/02/18/guide-to-poor-api-management/

I know someone found the downloads in a cache/archived version of that page, try to search the issues maybe you'll find. 

Just be aware that you'll be downloading binaries and you don't know if those binaries are the officials one.

**Comment by finnbrue** [comment]
Thanks a lot, i look at it.
